linera-views/src/dynamo_db.rs

```diff
@@ -1165,3 +1165,154 @@ impl IsResourceInUseException for SdkError<aws_sdk_dynamodb::error::CreateTableE
+
+        let context = LocalStackTestContext {
+            base_config,
+            endpoint,
+            _guard,
+        };
+
+        context.clear().await?;
+
+        Ok(context)
+    }
+
+    /// Creates an [`Endpoint`] using the configuration in the [`LOCALSTACK_ENDPOINT`] environment
+    /// variable.
+    fn load_endpoint() -> Result<Endpoint, Error> {
+        let endpoint_address = env::var(LOCALSTACK_ENDPOINT)
+            .with_context(|| {
+                format!(
+                    "Missing LocalStack endpoint address in {LOCALSTACK_ENDPOINT:?} \
+                    environment variable"
+                )
+            })?
+            .parse()
+            .context("LocalStack endpoint address is not a valid URI")?;
+
+        Ok(Endpoint::immutable(endpoint_address))
+    }
+
+    /// Creates a new [`aws_sdk_s3::Config`] for tests, using a LocalStack instance.
+    pub fn s3_config(&self) -> aws_sdk_s3::Config {
+        aws_sdk_s3::config::Builder::from(&self.base_config)
+            .endpoint_resolver(self.endpoint.clone())
+            .build()
+    }
+
+    /// Creates a new [`aws_sdk_dynamodb::Config`] for tests, using a LocalStack instance.
+    pub fn dynamo_db_config(&self) -> aws_sdk_dynamodb::Config {
+        aws_sdk_dynamodb::config::Builder::from(&self.base_config)
+            .endpoint_resolver(self.endpoint.clone())
+            .build()
+    }
+
+    /// Removes all stored data from LocalStack storage.
+    async fn clear(&self) -> Result<(), Error> {
+        self.remove_buckets().await?;
+        self.remove_tables().await?;
+
+        Ok(())
+    }
+
+    /// Removes all buckets from the LocalStack S3 storage.
+    async fn remove_buckets(&self) -> Result<(), Error> {
+        let client = aws_sdk_s3::Client::from_conf(self.s3_config());
+
+        for bucket in list_buckets(&client).await.unwrap_or_default() {
+            let objects = client.list_objects().bucket(&bucket).send().await?;
+
+            for object in objects.contents().into_iter().flatten() {
+                if let Some(key) = object.key.as_ref() {
+                    client
+                        .delete_object()
+                        .bucket(&bucket)
+                        .key(key)
+                        .send()
+                        .await?;
+                }
+            }
+
+            client.delete_bucket().bucket(bucket).send().await?;
+        }
+
+        Ok(())
+    }
+
+    /// Removes all tables from the LocalStack DynamoDB storage.
+    async fn remove_tables(&self) -> Result<(), Error> {
+        let client = aws_sdk_dynamodb::Client::from_conf(self.dynamo_db_config());
+
+        for table in list_tables(&client).await.unwrap_or_default() {
+            client.delete_table().table_name(table).send().await?;
+        }
+
+        Ok(())
+    }
+}
+
+/// Creates a basic client that can be used for tests.
+pub async fn create_dynamo_db_test_client() -> DynamoDbClient {
```

Comment:
I thought we were using a method for this now?

Reply:
There is a method, but for the `Store`.
The change could be done, but likely in another PR since the change would be huge.